dashboard/src/components/AppNew/AppNew.tsx

```diff
@@ -111,6 +135,11 @@ class AppNew extends React.Component<IAppNewProps, IAppNewState> {
         <form className="container" onSubmit={this.handleDeploy}>
```

Comment:
not really related to this PR, but while your on this view can we add the `margin-b-bigger` class to the form, right now there is no spacing underneath the form.

Reply:
fixed! i assumed this was specific to firefox :joy: